pallets/pallet-asset-switch/src/tests/force_set_switch_pair.rs

```diff
@@ -59,7 +59,11 @@ fn successful() {
 				status: SwitchPairStatus::Paused,
 			};
 			assert_eq!(switch_pair, Some(expected_switch_pair.clone()));
-			assert_total_supply_invariant(u64::MAX, expected_switch_pair.remote_asset_balance, &pool_account_address);
+			assert_total_supply_invariant(
```

Comment:
I assume, these are only fmt changes, right?

Reply:
Yeah but they're gone in the `try-runtime` PR: https://github.com/KILTprotocol/kilt-node/pull/674.